Doc/library/select.rst

```diff
@@ -327,30 +296,65 @@ Edge and Level Trigger Polling (epoll) Objects
    Create an epoll object from a given file descriptor.
 
 
-.. method:: epoll.register(fd[, eventmask])
-
-   Register a fd descriptor with the epoll object.
+.. method:: epoll.register(fd, eventmask=EPOLLIN | EPOLLOUT | EPOLLPRI)
 
+   Register a *fd* descriptor with the epoll object.
 
-.. method:: epoll.modify(fd, eventmask)
-
-   Modify a registered file descriptor.
+   *eventmask* is an optional bitmask describing the type of events you want
+   to check for.
 
 
 .. method:: epoll.unregister(fd)
 
    Remove a registered file descriptor from the epoll object.
 
 
-.. method:: epoll.poll(timeout=-1, maxevents=-1)
+.. method:: epoll.modify(fd, eventmask)
 
-   Wait for events. timeout in seconds (float)
+   Modify a registered file descriptor.
 
-   .. versionchanged:: 3.5
-      The function is now retried with a recomputed timeout when interrupted by
-      a signal, except if the signal handler raises an exception (see
-      :pep:`475` for the rationale), instead of raising
-      :exc:`InterruptedError`.
+
+The following constants can be used with
+:meth:`~epoll.register` and :meth:`~epoll.modify` methods.  See
+:manpage:`epoll(4)` for more information.
```

Comment:
I wonder if this table should be under `epoll.register`, similar to the section for `poll`.